apps/indexer/lib/indexer/helper.ex

```diff
@@ -724,8 +724,8 @@ defmodule Indexer.Helper do
   end
 
   @doc """
-    Sends HTTP GET request to the given URL and returns JSON response. Makes max three attempts and then returns an error in case of failure.
-    There is a timeout between attempts (increasing from 3 seconds to 20 minutes as the number of attempts increases).
+    Sends HTTP GET request to the given URL and returns JSON response. Makes max nine attempts and then returns an error in case of failure.
```

Comment:
if the count starts from 0, shouldn't it be 10?

```suggestion
    Sends HTTP GET request to the given URL and returns JSON response. Makes max 10 attempts and then returns an error in case of failure.
```

Reply:
Yes, you're right